ecosystem/typescript/sdk/src/aptos_client.ts

```diff
@@ -149,8 +149,8 @@ export class AptosClient {
       max_gas_amount: "1000",
       gas_unit_price: "1",
       gas_currency_code: "XUS",
-      // Unix timestamp, in seconds + 10 minutes
-      expiration_timestamp_secs: (Math.floor(Date.now() / 1000) + 600).toString(),
+      // Unix timestamp, in seconds + 10 seconds
+      expiration_timestamp_secs: (Math.floor(Date.now() / 1000) + 10).toString(),
```

Comment:
Somewhat worrying if transactions start taking longer than 10 seconds, but cest la vie.

Reply:
believe the blockchain is fast :) I think 30s is probably reasonable expectation